### PR TITLE
toggles also clickable by label

### PR DESCRIPTION
**Pull request by TimBuckers**
New feature (requested by Moritz)

### CODE REVIEW

**Comment by SunweiWang** [comment]
Nice, I checked all the labels, they are all clickable now. :+1:

**Comment by ClintonCao** [comment]
I've tested it and it works fine. Also I just noticed that the relative button and the color scale text shows a description when you hover over it. Nice job!  Can you maybe remove the logging/debugging info in the console? :stuck_out_tongue:

File: visualization/helperJSFiles/handleClicks.js [hunk 1]
```diff
@@ -21,6 +21,14 @@ function handleClickTreeMapTypeSat(value, checked) {
         }
     }
 }
+/**
+* Handles all clicks on label redirect to a click on the box
+*/
+function onClickLabel(idToggle) {
+	console.log(idToggle);
```
Comment (SunweiWang) [inline visualization/helperJSFiles/handleClicks.js]: NO, don't remove this, this looks super nice!!! @ClintonCao  :computer:

Comment (ClintonCao) [inline visualization/helperJSFiles/handleClicks.js]: Logging all the click information is nice to keep? Not sure if you want the user to see that :stuck_out_tongue:

**Comment by TimBuckers** [comment]
console log removed :)

**Comment by SunweiWang** [comment]
nice, I will merge this.

**Pull request merged**